### PR TITLE
Bring in styles and custom markup from the prototype

### PR DESCRIPTION
**Pull request by helenb**
### What is the context of this PR?

Ticket:  https://jira.ons.gov.uk/browse/CMS-226

- Update the linting rules inherited from wagtail, and fix some linting issues not present in the prototype
- Remove any custom font-sizes and most line-heights from the imported css, and use design system classes instead
- Fix the display of charts in the ons-embed block for now - we need a separate embed block ultimately
- Don't import customisations to the document-list styling or template
- Update the headers on the analysis and topic pages

This also fixes the issue raised in https://jira.ons.gov.uk/browse/CMS-223

### How to review

Check out the branch on your local build, and ensure the front-end files have been compiled. To do this on your local machine run the following:

`nvm use` - to ensure you are using the correct node version
`npm ci` - if you need to install the node modules
`npm run build` - to compile the files

### CODE REVIEW

File: cms/jinja2/templates/pages/topic_page.html [hunk 1]
```diff
@@ -2,20 +2,30 @@
...
     <div class="topic-header">
+        <div class="topic-header__circles" aria-hidden="true">
+            <div class="topic-header__circle-one"></div>
+            <div class="topic-header__circle-two"></div>
+            <div class="topic-header__circle-three"></div>
+        </div>
         <div class="ons-container">
             {% include "templates/components/navigation/breadcrumbs.html" %}
-            <p class="topic-header__doc-type">{{ _("Topic") }}</p>
+            <p class="topic-header__doc-type">Topic</p>
```
Comment (zerolab) [inline cms/jinja2/templates/pages/topic_page.html:12]:
```suggestion
            <p class="topic-header__doc-type">{{ _("Topic") }}</p>
```

File: cms/jinja2/templates/pages/analysis_page.html [hunk 2]
```diff
@@ -4,19 +4,20 @@
...
 {% from "components/text-indent/_macro.njk" import onsTextIndent %}
 
+
 {% block header_area %}
-    <div class="analysis-header">
+    <div class="analysis-header {% if page.updates %}analysis-header--flush{% endif %}">
         <div class="ons-container">
             {% include "templates/components/navigation/breadcrumbs.html" %}
-            <p class="analysis-header__doc-type">{{ _("Analysis") }}</p>
+            <p class="analysis-header__doc-type ons-u-fs-l">{{ _("Analysis") }}</p>
```
Comment (zerolab) [inline cms/jinja2/templates/pages/analysis_page.html:12]: The designs (and prototype) now have 

```suggestion
            <p class="analysis-header__doc-type ons-u-fs-l">{{ _("Statistical article") }}</p>
```

should we stick to it?

File: cms/jinja2/templates/base_page.html [hunk 3]
```diff
@@ -29,23 +30,21 @@
...
+        {# This will contain the breadcrumbs on individual templates #}
+        {% block header_area %}
+        {% endblock %}
         <div class="ons-page__container ons-container{{ containerClasses }}">
             <div class="ons-grid">
                 <div class="ons-grid__col ons-col-{{ pageColNumber }}@m {{ pageColClasses }}">
                     {# This is the main tag in the base template in the design system #}
                     <div class="ons-page__main {{ mainColClasses }}">
                         {% block main %}{% endblock %}
+                        {{ onsBackToTop() }}
```
Comment (zerolab) [inline cms/jinja2/templates/base_page.html:47]:
```suggestion
                        {{ onsBackToTop() }}{# used in small viewports #}
```

**Review by zerolab — COMMENTED** [review]
Everything is looking sooo much better. Left a few suggestions.

**Comment by helenb** [comment]
> Everything is looking sooo much better. Left a few suggestions.

That was speedy! I have a few more to dos before this is totally ready though.

File: cms/jinja2/templates/pages/analysis_page.html (hunk 2 above)
Comment (helenb) [reply cms/jinja2/templates/pages/analysis_page.html:12]: I was going by the figma file - and I think I recall a November prototype conversation where it was agreed it should be 'Statistical article' - but I will check.

File: cms/jinja2/templates/pages/topic_page.html (hunk 1 above)
Comment (helenb) [reply cms/jinja2/templates/pages/topic_page.html]: Good spot - fixed.

File: cms/jinja2/templates/base_page.html (hunk 3 above)
Comment (helenb) [reply cms/jinja2/templates/base_page.html]: Thanks - have added the comment

File: cms/jinja2/templates/pages/analysis_page.html (hunk 2 above)
Comment (helenb) [reply cms/jinja2/templates/pages/analysis_page.html:12]: Sorry - I misunderstood the diff there, and thought I already had 'Statistical article' as the text - however @MebinAbraham has suggested on slack that we leave it as 'Analysis' and have a separate card for updating to 'Statistical article'.

**Review by zerolab — APPROVED** [review]
LGTM

File: cms/jinja2/templates/pages/analysis_page.html (hunk 2 above)
Comment (zerolab) [reply cms/jinja2/templates/pages/analysis_page.html:12]: Works for me

**Review by MebinAbraham — APPROVED** [review]
Refreshing 🎉

**Review by sanjeevz3009 — COMMENTED** [review]
Reviewed!

**Pull request merged**